README.md

```diff
@@ -0,0 +1,23 @@
+# matlab-travis-lang-test [![Build Status](https://staging.travis-ci.org/mathworks-continuous-integration/matlab-travis-lang-test.svg?branch=master)](https://staging.travis-ci.org/mathworks-continuous-integration/matlab-travis-lang-test)
+
+Repo for testing continuous integration support for MATLAB using Travis CI.
+
+## MATLAB? Using Travis CI?
```

Comment:
Do we need the question marks? :D

Reply:
Nope; I thought was being "cool" 😂 
I changed it to a more neutral "How it works:" 🤔